table/src/main/java/tech/ydb/table/Session.java

```diff
@@ -72,6 +75,12 @@ CompletableFuture<Result<DataQueryResult>> executeDataQuery(
         Params params,
         ExecuteDataQuerySettings settings);
 
+    CompletableFuture<Result<ResultSetReader>> readRows(String pathToTable, List<StructValue> keys,
```

Comment:
Its not quite obvious what "StructValue" here should contain. Better provide some comments here.
Could be taken from here: https://github.com/ydb-platform/ydb-api-protos/blob/master/protos/ydb_table.proto#L1056

Reply:
Thanks, added comment